docs/ABC_of_pull_requests.md

```diff
@@ -58,29 +70,29 @@ Next is the code review itself. Behavior is one thing, code quality is another.
 
 ***To do when:***
 
-- [ ] The code touches important core parts, is not well tested and may have side effects
+- [ ] The code touches important core parts, is not well tested and may have side effects.
 - [ ] The author specifically mentioned to test some specific thing.
 - [ ] There is some behavior change that requires validation by maintainers.
 
 If functional review is required, here are the steps to follow:
 
 - [ ] **Get the code**: There are instructions at the bottom of the PR to get the code. Click on the `command line instructions` link beside the green Merge button. See screenshot below. Refuse if there are conflicts with `dev`.
 
-![Get the code](images/PRs/getTheCode.png)
```

Comment:
You have to keep the ! for the images to display. Have you seen the PR I did on your branch? https://github.com/mahdiguermache/transition/pull/3  You can merge it to your branch and it will be automatically added to the current PR (I think). If you don't, I'll do a PR here right after merging this one.